src/client/components/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Paper, Button, FormControl, OutlinedInput, InputLabel, Typography } from '@material-ui/core';
+import './LoginForm.css';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& > *': {
+      margin: theme.spacing(1),
+    },
+  },
+}));
+
+function LoginForm() {
+  const classes = useStyles();
+  const [login, setLogin] = React.useState('');
+  const [password, setPassword] = React.useState('');
+  return (
+    <Paper elevation={3} className="form-box">
+      <form className={classes.root}>
+        <Typography gutterBottom variant="h5" component="h2">
+          Login
+        </Typography>
+        <FormControl variant="outlined">
+          <InputLabel htmlFor="component-outlined">Login</InputLabel>
+          <OutlinedInput
+            id="input-login"
+            value={login}
+            label="Login"
+            onChange={(e) => setLogin(e.target.value)}
```

Comment:
to prevent recreating function every rerender better to use: `const handleLoginChange = useCallback(({ target: { value } }) => setLogin(value), []);`